taskcluster/configs/config.ci.yml

```diff
@@ -96,6 +96,7 @@ datasets:
     - sacrebleu_aug-upper_wmt19
   test:
     - flores_devtest
+    - tmx_aug-short_pontoon
```

Comment:
I hope it's not too slow for CI

Reply:
The dataset task, which also adds alignments for 1000 lines flores takes 6m: https://firefox-ci-tc.services.mozilla.com/tasks/BjKp-oXGQsOVprA8Gv0DyQ/runs/0/logs/public/logs/live.log. If it's in the same ballpark, we're good. Probably the dataset downloading might take longer.

I think not everything has to be included in CI. 8m GPU task is fine. If it takes longer like 20m, it's probably better to skip it in CI as it should be fast and cheap. 

Agreed on optimizations overall. 1000k was more about making the sample more statistically representative to follow other evals. We don't have to optimize it. If it's slow, we can remove it from CI.

Reply:
@ZJaume do we understand why adding alignments in the dataset tasks that run on a CPU with the same method takes 1 minute? https://firefox-ci-tc.services.mozilla.com/tasks/cTMeVhoyTHuQYWTglcttJg/runs/0/logs/public/logs/live.log

Reply:
I think we can remove `tmx_aug-short_pontoon` from CI, but since the metric will be calculated for all datasets, as intended by this design, any overhead will be multiplied by 1000s of runs. So I'd make sure it works efficiently enough and there are no bugs at least.